railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_8_2.rb.tt

```diff
@@ -9,6 +9,19 @@
 # Read the Guide for Upgrading Ruby on Rails for more info on each option.
 # https://guides.rubyonrails.org/upgrading_ruby_on_rails.html
 
+# Active Job
+
+###
+# Defer job enqueueing until after the database transaction commits.
+#
+# When enabled, jobs enqueued inside a transaction wait until the transaction
+# commits before being dispatched to the queue. This prevents jobs from
+# operating on rolled-back data.
+#++
+# ActiveSupport.on_load(:active_job) do
+#   ActiveJob::Base.enqueue_after_transaction_commit = true
+# end
```

Comment:
I think it's a bit of an anti-pattern to put initializer code here, typically we only put `Rails.application.config` afaik

Maybe it's ok to keep the config for this since the previous deprecation cycle is concluded, and we give people at least a MINOR version to adopt the change.

Reply:
Right, this is the ugly code foisted upon apps referenced above. I'm changing it back to global config and un-deprecating.